tests/llama_index/test_llama_index_evaluate.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+import pytest
+
+import mlflow
+from mlflow.metrics import latency
+
+
+@pytest.mark.parametrize("engine_type", ["query", "chat"])
+def test_llama_index_evaluate(engine_type, single_index):
+    with mlflow.start_run():
+        model_info = mlflow.llama_index.log_model(
+            index=single_index,
+            engine_type=engine_type,
+            artifact_path="llama_index",
+        )
+
+    pyfunc_model = mlflow.pyfunc.load_model(model_info.model_uri)
+
+    data = pd.DataFrame(
```

Comment:
Can we verify with an additional parametrization config that the behavior is the same for a single-row DataFrame as well, just in case there are any positional-based access of the data within the call stack? Pandas does different collections generation (IIRC) based on whether a DF has a single row vs. multiple rows.